test/pull/expected-stdout

```diff
@@ -11,12 +11,12 @@ INFO		+  C bar/extractor/keboola.ex-db-mysql/with-rows
 INFO		+  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/disabled
 INFO		+  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/test-view
 INFO		+  R bar/extractor/keboola.ex-db-mysql/with-rows/rows/users
-INFO		+  C bar/other/keboola.variables/python-transformation-shared-code-variables
-INFO		+  R bar/other/keboola.variables/python-transformation-shared-code-variables/rows/default-values
-INFO		+  C bar/other/keboola.variables/shared-code-variables
 INFO		+  C bar/transformation/keboola.python-transformation-v2/python-transformation
 INFO		+  C bar/transformation/keboola.python-transformation-v2/python-transformation-shared-code
 INFO		+  C bar/transformation/keboola.snowflake-transformation/snowflake-transformation
+INFO		+  C bar/variables
```

Comment:
Oprava sa hned prejavila v `pull` teste, kde sa tento naming pouzil.
Este nie je funkcne napojenie na nadradeny config, preto sa `variables` vygenerovali priamo do adresara branche.

V dalsi PRs sa to fixne, a `variables` budu pod configom, ktory ich pouziva.